java/s2i/run

```diff
@@ -10,4 +10,14 @@ JAVA_OPTIONS="${JAVA_OPTIONS:+${JAVA_OPTIONS} }$(/opt/jolokia/jolokia-opts)"
 # Temporary options variable until the harmonization hawt-app PR #5 has been applied (hopefully)
 JVM_ARGS="${JVM_ARGS:+${JVM_ARGS} }${JAVA_OPTIONS}"
 export JAVA_OPTIONS JVM_ARGS
-exec ${DEPLOYMENTS_DIR}/bin/run.sh
\ No newline at end of file
+
+if [ -f "${DEPLOYMENTS_DIR}/bin/run.sh" ]; then
+    echo "Starting Hawtapp Application ..."
+    exec ${DEPLOYMENTS_DIR}/bin/run.sh
+elif [ -f "${DEPLOYMENTS_DIR}/run-java.sh" ]; then
+    echo "Starting Java Application ..."
+    exec ${DEPLOYMENTS_DIR}/run-java.sh
```

Comment:
run-java.sh can cope with hawt-app app structure as well, btw (i.e. using a `classpath` file for creating the classpath).